### PR TITLE
Supafeature

### PR DESCRIPTION
**Pull request by sanarise**
Some comment

### CODE REVIEW

File: feature.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Line 1
+Line 2
```
Comment (sanarise-pr) [inline feature.txt:2]: It's awesome!

File: feature.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Line 1
```
Comment (sanarise-pr) [inline feature.txt:1]: Hmm...

Comment (sanarise) [reply feature.txt:1]: wat!

Comment (sanarise-pr) [reply feature.txt:1]: Why not 0?

Comment (sanarise-pr) [reply feature.txt:1]: ...

**Review by sanarise-pr — COMMENTED** [review]
finish

**Review by sanarise-pr — APPROVED** [review]

**Pull request merged**